src/Speckle.Objects/Other/RawEncoding.cs

```diff
@@ -0,0 +1,23 @@
+using Speckle.Sdk.Models;
+
+namespace Speckle.Objects.Other;
+
+/// <summary>
+/// Keeps track of a raw-encoded object in a native supported format. see <see cref="RawEncodingFormats"/>
+/// </summary>
+[SpeckleType("Objects.Other.RawEncoding")]
+public class RawEncoding : Base // note: at this stage, since we're using this for extrusions and subds the name doesn't make sense anymore
+{
+  public required string format { get; set; }
+  public required string contents { get; set; }
+
+  public RawEncoding() { }
+}
+
+/// <summary>
+/// Supported encoding types "strongly" typed strings. This needs to match the extension of the file format.
+/// </summary>
+public static class RawEncodingFormats
+{
+  public const string RHINO_3DM = "3dm";
+}
```

Comment:
We really should have `Enum` serialisation to `string` and back to avoid having to create this constant strings. Enums are just nicer overall.

@adamhathcock wdyt?

Reply:
We should come up with something better but no reason to block this for now.

I'm in favor of enums with source gen :)